ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -1590,6 +1590,7 @@ tests:
     env:
       E2E_RUN_TAGS: '@nutanix-ipi'
       FIPS_ENABLED: "true"
+      RT_ENABLED: "true"
```

Comment:
please also add `-rt-` keyword to the job name to show it's a realtime kernel test

Reply:
fixed, thanks.